src/labels/CustomLabels.labels

```diff
@@ -2107,6 +2107,22 @@ If you&apos;re not sure how to resolve these errors, post a message in the Nonpr
         <shortDescription>Title of the Schedule Visualizer Component</shortDescription>
         <value>Upcoming Installments</value>
     </labels>
+    <labels>
+        <fullName>RD2_StatusMustBeMapped</fullName>
+        <categories>Recurring-Donations, Error, Health Check</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Message when Enhanced RD status is not mapped</shortDescription>
+        <value>The Status &quot;{0}&quot; hasn't been mapped to a State.</value>
+    </labels>
+    <labels>
+        <fullName>RD2_StatusMustBeMappedSolution</fullName>
+        <categories>Recurring-Donations, Error, Health Check</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Solution when Enhanced RD status is not mapped</shortDescription>
+        <value>Go to NPSP Settings | Recurring Donations | Status to State Mappings, or contact your administrator for help.</value>
+    </labels>
```

Comment:
@sam-knox Please review the 2 custom labels, it's the same one from [W-040565](https://github.com/SalesforceFoundation/NPSP/pull/5392) but split into error and solution part
![Screen Shot 2020-05-01 at 12 37 23 PM](https://user-images.githubusercontent.com/57271120/80836023-ca1ebc80-8ba8-11ea-85e8-9fa164797240.png)

Reply:
![Screen Shot 2020-05-01 at 4 17 47 PM](https://user-images.githubusercontent.com/57271120/80848200-54761900-8bc7-11ea-8010-b7bebfcdab94.png)
@sam-knox Thanks for the review. The label is updated.